etc/genome/spec/sys_lock_exit_code.yaml

```diff
@@ -0,0 +1,3 @@
+---
+env: XGENOME_SYS_LOCK_EXIT_CODE
+default_value: ' '
```

Comment:
It might be nice to add:
```yaml
validator:
  - Numeric
```

It's lame that this has to have a default value to avoid crashing, and if we did validate it we'd be stuck with using something like 0 as a special value.